app/controllers/foods_controller.rb

```diff
@@ -82,4 +79,18 @@ def upload_image
      end
      @food_image_url
   end
+
+  def save_food
+    @food[:food_image_file_name] = @food_image_url
+    if food_params[:sales][:js]
+      food = food_params[:sales]
+      food_status = (food["status"] == "true") ? true : false
+      @food.update(sales: {price: food["price"].to_f, percentage: food["percentage"].to_f, status: food_status})
```

Comment:
Line is too long. [112/80]<br>Space inside { missing.<br>Space inside } missing.